src/configfile.ts

```diff
@@ -426,6 +458,25 @@ function parseBrowserObject(config: BrowserConfigs): BrowserConfig {
   if ('preferences' in config && config.preferences) {
     parsed.preferences = config.preferences;
   }
+  if ('trace' in config && config.trace !== undefined) {
+    if (config.trace === false) {
+      // Do nothing
```

Comment:
TypeScript requires I handle both boolean types in order for the object type to correctly compile below.